msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
```

Comment:
See https://github.com/git-for-windows/build-extra/compare/master...dscho:msi :smile: